codebuild/release/prod-release.yml

```diff
@@ -9,21 +9,33 @@ env:
 
 phases:
   install:
+    commands:
+      - pip install tox
+      - pip install --upgrade pip
     runtime-versions:
       python: latest
+  pre_build:
```

Comment:
What's the functional difference between `build` and `pre_build`?

Reply:
To be honest I haven't found a clear description of any functional differences. The docs provide some examples of what you could do in each one, but there doesn't seem to be any concrete. My grouping here is fairly arbitrary, so I'm open to suggestions.

Reply:
This is fine, it looks like `pre_build/build/post_build` are really just to help organize the phases, and this seems to fit well.